cmd/tempo-serverless/lambda/Dockerfile.proxy

```diff
@@ -0,0 +1,44 @@
+# 
```

Comment:
if this is intended to be a proxy image only we could clean it up quite a bit. there's no need to put the lambda binary in here.

also, i'm not seeing changes to the e2e tests?

Reply:
I'll be looking at cleaning up the proxy file, and maybe relocating it.  I'm thinking we want to produce a proxy docker image for the e2e environment.